sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Release History
 
 ## 1.0.0b5 (2021-03-09)
-- Updated `azure-communication-chat` version.
+- Added support for communication identifiers instead of raw strings.
```

Comment:
Please confirm if they are breaking changes and if so include that sub-heading

Reply:
@ankitarorabit Done

Reply:
Thank you!